patches/third_party-blink-renderer-core-context_features-context_feature_settings.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/context_features/context_feature_settings.h b/third_party/blink/renderer/core/context_features/context_feature_settings.h
+index 9c9a289262c01daa45e24d4cb553b7d90797f7b8..de4e8ced39169f817f6c22842013557198630137 100644
+--- a/third_party/blink/renderer/core/context_features/context_feature_settings.h
++++ b/third_party/blink/renderer/core/context_features/context_feature_settings.h
+@@ -35,6 +35,7 @@ class CORE_EXPORT ContextFeatureSettings final
+   bool isMojoJSEnabled() const { return enable_mojo_js_; }
+ 
+   void Trace(Visitor*) const override;
++  BRAVE_CONTEXT_FEATURE_SETTINGS_H_
```

Comment:
this made sense https://github.com/brave/brave-core/pull/6554/commits/42f1e5a016fb40ce23e347693355ee56dbaa3b8b#diff-4a7ab71a64ac5712bb35fb91782847039cf2def6ee9c3d64a7efa66562a99acfL225
but a patch doesn't make sense. I think we're missing something here

Reply:
Not a lot of examples how to do this:
<pre>
$ git grep -n ContextEnabled -- '*.idl'
third_party/blink/renderer/bindings/tests/idls/core/test_interface.idl:44:    ContextEnabled=RuntimeFeature,
third_party/blink/renderer/core/frame/reporting_observer.idl:10:    ContextEnabled=ReportingObservers,
third_party/blink/renderer/core/mojo/mojo.idl:22:    ContextEnabled=MojoJS,
third_party/blink/renderer/core/mojo/mojo_handle.idl:8:    ContextEnabled=MojoJS,
third_party/blink/renderer/core/mojo/mojo_watcher.idl:8:    ContextEnabled=MojoJS,
</pre>

I just followed the example of MojoJS. Perhaps this doesn't need to be ContextEnabled necessarily @jumde?